badgecheck/reducers/graph.py

```diff
@@ -1,16 +1,45 @@
+import copy
+
 from ..actions.action_types import ADD_NODE, UPDATE_NODE
 
+current_node_id = -1
```

Comment:
minor: maybe we want to name the counter something other than 'node_id' as we use the term elsewhere to refer to the node.id properties value, whereas this counter is only a part of that.